### DISCUSSION TITLE
Enable/Disable next and previous button in the wizard

### DISCUSSION THREAD

**Discussion by sh4msi** [filamentphp/filament#5030] [Help]
How can I enable/disable the next and previous button in the wizard?

**Comment by ssatz** [comment]
Can anybody here have a solution for this?

**Reply by danharrin** [reply]
Please provide more detail on what the use case is

**Reply by ssatz** [reply]
Use Case: Verify User Identity and Add Bank Account. Reference image attached
![image](https://user-images.githubusercontent.com/6892242/230605709-bfce1cb9-1dca-4638-a68f-68d65edbe3cc.png)

- Verify Email - By Sending OTP (If OTP verified, then user no need to go back)
- Verify Personal Identity - By Sending OTP to Mobile (If OTP verified, then user no need to go back)
- Add Bank Account

**Reply by danharrin** [reply]
We don't have a way to disable the previous button atm but you could PR it :)

**Reply by sinkovmikhail** [reply]
This works for me.
https://github.com/filamentphp/filament/discussions/9942#discussioncomment-7702242

**Comment by fvanzeijl** [comment]
> How can I enable/disable the next and previous button in the wizard?

You can do by overriding the wizard-view.
Just copy file
`vendor/filament/forms/resources/views/components/wizard.blade.php` 
to 
`views/vendor/forms/components`

**Reply by danharrin** [reply]
Please dont do that, https://filamentphp.com/blog/publishing-views-in-laravel

**Reply by fvanzeijl** [reply]
In general I totally agree with you. 
But: The fact I can still override several filament-views stands totally in contrast to the article you provided.
Imho its legal to override views as long as the devs be aware of that it could break something on updated (so your provided link is a good point).

Anyways. Let's see whats v3 brings!

**Reply by diogogomeswww** [reply]
I have exactly this use case. I need to confirm the data before passing to the next step.

Is it possible conditionally hide the next action on the v3 wizard?

**Reply by leandrocfe** [reply]
you can use `->beforeValidation()` or `->afterValidation()`

**Reply by ssatz** [reply]
But you can’t disable/enable previous/next button

**Comment by rozaqabdul656** [comment]
->previousAction(
                        fn (Action $action) => $action->disabled(true),
                    )
                    
                    use this u can disable
